### PR TITLE
[add] reverse array function

### PR DESCRIPTION
**Pull request by iSumitBanik**
This PR will add a new algorithm in this repo which function is to reverse an Array.

Issue #41 

CHANGELOG

- Added the ReverseArray Contract under src/Maths
- Added the corresponding foundry test under src/Maths/tests
- Added the Contract hyperlink in the README

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/Solidity/pull/54"></a>

### CODE REVIEW

File: src/Maths/ReverseArray.sol [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    */
+    function reverse(uint256[] memory _array)
+        external
+        pure
+        returns (uint256[] memory)
+    {
+        uint256 end = (_array.length) - 1;
+        return _reverseArray(_array, 0, end);
+    }
+}
```
Comment (Panquesito7) [inline src/Maths/ReverseArray.sol:45]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

Comment (iSumitBanik) [reply src/Maths/ReverseArray.sol]: Added the missing endline.

**Review by mkubdev — DISMISSED** [review]
Hi, thanks a lot for your contribution. This look good to me ! ⚡

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**